### ISSUE TITLE
Why status of app not changed after downloadin'

### ISSUE THREAD

**Issue by Ygarr** [NeoApplications/Neo-Store#272]
After update you can tap "update" infinitely and app will be downloading every time and not installing. NeStore ver 9.3

**Comment by abdulrahman1s** [comment]
I had the same issue 🙁

**Comment by ahmedmahm0ud** [comment]
Same here ver. 0.9.3

**Comment by magnus-ISU** [comment]
This happened to me on one specific app. I downloaded it like 10 times. Then I went to the normal Fdroid app and downloaded the same app and the apk was apparently corrupt so it wouldnt install. I suspect NeoChat just isn't showing error messages if errors happen.

**Comment by kalekad** [comment]
Maybe it's a different error, but sometimes when the installation fails, this error message appears:

Unknown error.
INSTALL_FAILED_INVALID_APK: Session:
1479548092. No packages staged in /d...

**Comment by Ygarr** [comment]
After recent update I see: the issue is fixed

**Comment by revuwa** [comment]
> After update you can tap "update" infinitely and app will be downloading every time and not installing. NeStore ver 9.3

I had/have absolutely the same behavior, started with app version 0.9.3 and still exists in 0.9.4 @ Android 12, and meanwhile updated to Android 13. Would love to see that fixed.
The only workaround I figured out is to completely reset NeoStore (settings -> storage & cache -> clear storage). After that I can download install some apps, as expected, but the bug comes back after a few app installations, and then I can't install anything, until I fully reset the app again. WiFi or cellular doesn't matter.

**Comment by revuwa** [comment]
PS: a second workaround could be to uninstall the current NeoStore, reinstall the previous (working) version 0.9.2, and set "ignore all new versions" inside the NeoStore app screen, until there're new insights in this. 
It's a dirty workaround and no recommendation, but possibly somebody is as desperate as I am.

**Comment by revuwa** [comment]
Update from my side. I reinstalled v0.9.5 completely. The behavior doesn't changed, and after a few downloads, the install-request doesn't appear anymore (as in the versions before), BUT, since this version it the workaround 'clear cache' seems to do the trick for me. Tested this with (a clean install of) v0.9.6, too.
If the problem came up, I cleared the cache over: settings of the app -> storage & cache -> clear cache, (exactly one time after clean install), and never got the problem again. With other words, my problem with versions >=0.9.3 has gone.
If someone asks, the clear cache method doesn't worked for me on versions <=0.9.4...
